pkg/operator/ceph/cluster/controller.go

```diff
@@ -550,6 +564,30 @@ func (c *ClusterController) onUpdate(oldObj, newObj interface{}) {
 		}
 		return
 	}
+
+	// Display success after upgrade
+	if versionChanged {
+		versions, err := client.GetCephVersions(c.context, cluster.Namespace)
```

Comment:
Could this code block be factored out to a `PrintVersionStatus()` or similar method? This method is getting really long already.